urlchecker/client/check.py

```diff
@@ -89,17 +94,23 @@ def main(args, extra):
         delete_repo(path)
 
     # Case 1: We didn't find any urls to check
-    if not check_results['failed'] and not check_results['passed']:
+    if not check_results["failed"] and not check_results["passed"]:
```

Comment:
@SuperKogito I hope this better matches what you had in mind:

1. If there aren't any passing or failing urls, we exit with code 0
2. If there are failed urls, regardless of force pass, we print them for the user in red
3. Then if there are failed and there is no force pass, we exit with error
4. Finally, we exit with 0 (the last condition given that there are either failed urls with force pass, or there were none) but we modify the message printed so the user knows if it was a conditional pass.